cmd/kubeadm/app/cmd/upgrade/plan_test.go

```diff
@@ -436,30 +443,26 @@ Note: Before you can perform this upgrade, you have to update kubeadm to v1.19.3
 
 _____________________________________________________________________
 
+No information available on component configs.
 `),
 		},
 	}
 	for _, rt := range tests {
 		t.Run(rt.name, func(t *testing.T) {
 			rt.buf = bytes.NewBufferString("")
-			outputFlags := newUpgradePlanPrintFlags(output.TextOutput)
+			outputFlags := output.NewOutputFlags(&upgradePlanTextPrintFlags{}).WithTypeSetter(outputapischeme.Scheme).WithDefaultOutput(output.TextOutput)
 			printer, err := outputFlags.ToPrinter()
 			if err != nil {
 				t.Errorf("failed ToPrinter, err: %+v", err)
 			}
 
-			// Generate and print upgrade plans
-			for _, up := range rt.upgrades {
-				plan, unstableVersionFlag, err := genUpgradePlan(&up, rt.externalEtcd)
-				if err != nil {
-					t.Errorf("failed genUpgradePlan, err: %+v", err)
-				}
-				printUpgradePlan(&up, plan, unstableVersionFlag, rt.externalEtcd, rt.buf, printer)
-			}
+			plan := genUpgradePlan(rt.upgrades, nil, rt.externalEtcd)
```

Comment:
instead of nil for the second arg, please have at least one test case where the component configs are available.